src/routes/docs/products/auth/team-invites/+page.markdoc

```diff
@@ -91,7 +91,7 @@ val response = teams.createMembership(
 ```
 {% /multicode %}
 
-## Accept invitations {% #accept-invitations %}
+# Accept invitations
```

Comment:
I think it makes sense for this to be nested as this is client only.

Reply:
Right. Done